### PR TITLE
Feature/mobile menu

### PR DESCRIPTION
**Pull request by John-Kibirige**
### Activities  carried out 
- Create an animation when the hamburger menu is clicked using the toggle method in javascript
- Modify the styles in order to implement the  menu design
- Add relevant linters and correct any linter errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,27 +20,40 @@
...
+          </li>
+          <li>
+            <a href="#about">About</a
+            ><img src="./images/chevron.svg" alt="chevron" />
+          </li>
+          <li>
+            <a href="#contact">contact</a
+            ><img src="./images/chevron.svg" alt="chevron" />
+          </li>
+        </ul>
```
Comment (codecaiine) [inline index.html:41]: - [ ] Good job on your menu work :+1: but there are some issues you need to address :cry: Please try to adjust your mobile menu `background-image` so can you can see all the menu options and options right arrow very well. Right now it is a little bit different from the Figma template. :+1: 

- [ ] I would like to recommend you work on **the close (X) button**, Please try to adjust the size and use the same color  in order to make it similar to the Figma template :+1:  


### Your project
Screenshot from 2022-07-19 09-47-08

### Figma template
Screenshot from 2022-07-19 09-46-13

- [ ] Well done, when I click on a menu option, it leads me to the part mentioned (example: **About/Portfolio/contact**) :100: But it's impossible for me to scroll up or down afterward. I have to refresh the page, please fix this to give the user a pleasant user experience. :smile: 

https://www.loom.com/share/ed8f08191c74406e89930a393a3749e0

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Descriptive Pull request :heavy_check_mark: 
- Good use of Gitflow :heavy_check_mark: 
- Commits are meaningful :heavy_check_mark: 
- Clean code :heavy_check_mark:  

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codecaiine — APPROVED** [review]
Hi @Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Descriptive Pull request :heavy_check_mark: 
- Good use of Gitflow :heavy_check_mark: 
- Commits are meaningful :heavy_check_mark: 
- Clean code :heavy_check_mark:  
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by John-Kibirige** [comment]
Thank you @codecaiine  for giving me such helpful comments and reviews.

**Pull request merged**